ha/core/event_analyzer/watcher/watcher.py

```diff
@@ -51,6 +51,7 @@ def __init__(self, consumer_id: int, message_type: str, consumer_group: str,
         self.parser = event_parser
         self.subscriber = subscriber
         self._validate()
+        # TBD : Call MessageBus.init() from here if this watcher needs to be used in HA. Reference EOS-26999
```

Comment:
Is this not used code so far? Instead of a comment you may want to add code and forgo testing with comment, mentioning testing pending.

Reply:
On another thought, I think we can stay at comment only if this is unused code, and not spend any time here. Resolving this comment.